kernel/api/src/main/java/org/sakaiproject/content/api/ContentHostingService.java

```diff
@@ -1940,6 +1940,30 @@ public ContentCollectionEdit addCollection(String collectionId, String name)
 		throws PermissionException, IdUnusedException, IdUsedException, 
 				IdLengthException, IdInvalidException, TypeException;
 
+	/**
+	 * @param collectionId
+	 * @param name
+	 * @param limit
```

Comment:
I know the previous comment didn't have any description on what these param's meant but could you add a some text for why someone would use limit over just using the 2 parameter method? Maybe the 2 parameter one should really also just be calling this one with a value of 1 rather than this method calling the 2 parameter method?